src/main/webapp/event-info-page/eventInfo.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
nit: Could set a title for the page, such as <title>Event Info</title>, unless you're planning to make the title change according to the specific event the user clicked on.

Reply:
Good point. Thank you!